app/views/two_factor_authentication/sms_opt_in/error.html.erb

```diff
@@ -35,4 +31,6 @@
       ].select(&:present?),
     ) %>
 
-<%= render 'shared/cancel', link: sign_out_path %>
```

Comment:
- Most IDV-specific shared partials are under `shared/idv`, so at a glance I would expect this shouldn't assume IDV and work elsewhere. In fact, it seems like it's already used in quite a few other places, such as account pages?
- Otherwise might be nice to use [`PageFooterComponent`](https://github.com/18F/identity-idp/blob/main/app/components/page_footer_component.rb) to handle the border/padding/margin consistency.